src/orga/getting-in-touch.rst

```diff
@@ -6,13 +6,15 @@ Getting in Touch
 The community at conda-forge would be happy to connect with you. We have got several ways for you to communicate with us, primarily through issue trackers, mailing lists, and real-time chat.
 
 Issue Trackers
------------------
+---------------
 
-The main issue trackers that you will interact with are
+The main issue trackers that you will interact with are: 
 
-* `staged-recipes <https://github.com/conda-forge/staged-recipes/issues>`__: You'll use staged-recipes to create a new conda package on conda-forge
-* `Our main docs repo <https://github.com/conda-forge/conda-forge.github.io/issues>`__: You'll use this repo as a catch-all for issues where you're not sure where else to put them
-* `Our enhancement proposals repo <https://github.com/conda-forge/cfep/issues>`__: You'll use the enhancement proposals repo if you're interested in substantially changing the way conda-forge operates.
+* `staged-recipes <https://github.com/conda-forge/staged-recipes/issues>`__: You can use staged-recipes repository to create a new conda package on conda-forge.
+* `conda-forge.github.io repository <https://github.com/conda-forge/conda-forge.github.io/issues>`__: You can use this repository to raise policy issues and as a catch-all for other issues where you’re unsure where to put them.
+* If you want to file an issue about an individual package, go to that pacakge's feedstock repository. So, say if you want to raise an issue for the package ``unzip``, go to its feedstock repository `here <https://github.com/conda-forge/unzip-feedstock/issues>`__. You can search for the package you wish to open an issue for `here <https://github.com/conda-forge/feedstocks/tree/main/feedstocks>`_
```

Comment:
Replace "So, say" with "For example,".

"For example, if you want to raise an issue..."